Amethyst/Managers/WindowManager.swift

```diff
@@ -29,33 +27,45 @@ enum MouseState {
     case dragging
     case moving(window: SIWindow)
     case resizing(screen: NSScreen, ratio: CGFloat)
-    case doneDragging(atTime: NSDate)
+    case doneDragging(atTime: Date)
+}
+
+protocol MouseStateKeeperDelegate: class {
+	func focusedScreenManager() -> ScreenManager?
```

Comment:
Minor, but this shouldn't be tab-indented.

Reply:
I agree.  Not sure why Xcode is doing that.